src/components/Footer.vue

```diff
@@ -1,39 +1,42 @@
 <template>
   <footer class="app-footer" :class="{ transparent: itsTransparent }">
-    <div class="container">
-      <div class="skulls-wrap">
-        <img src="@/assets/images/skulll.svg" alt="" class="skulls-img img-1" />
-        <img src="@/assets/images/skulll.svg" alt="" class="skulls-img img-2" />
-      </div>
-      <nav class="footer-nav">
-        <!-- <router-link :to="{ name: 'Docs' }" class="nav-link">Docs</router-link>
-        <router-link :to="{ name: 'Tech' }" class="nav-link">Tech</router-link> -->
-        <router-link :to="{ name: 'Liquidations' }" class="nav-link"
-          >Liquidations</router-link
-        >
-      </nav>
-      <div class="links-wrap">
-        <a
-          target="_blank"
-          href="https://medium.com/abracadabra-money"
-          class="link-item"
-        >
-          <img src="@/assets/images/medium-pixel.svg" alt="" />
-        </a>
-        <a
-          target="_blank"
-          href="https://twitter.com/MIM_Spell"
-          class="link-item ml"
-        >
-          <img src="@/assets/images/twitter-pixel.svg" alt="" />
-        </a>
-      </div>
-    </div>
+    <SocialLink
+      :textBtn="textBtn"
+      :discord="discord"
+      :twitter="twitter"
+      :medium="medium"
+    />
```

Comment:
according to the task SocialLink component should have only two props: `data` and `link`, but your component has 4 props. It is wrong.